fluent-bundle/src/bundle.rs

```diff
@@ -185,6 +368,43 @@ impl<'bundle> FluentBundle<'bundle> {
         None
     }
 
+    /// Formats both the message value and attributes identified by `message_id`
+    /// using `args` to provide variables.
```

Comment:
I'd like to add a sentence here like : "This method is useful for cases where the customer wants to apply a compound message onto a UI element."